Source/WebCore/platform/graphics/SourceBufferPrivateClient.h

```diff
@@ -43,6 +43,25 @@ class MediaDescription;
 class PlatformTimeRanges;
 class VideoTrackPrivate;
 
+struct SourceBufferEvictionData {
+    uint64_t contentSize { 0 };
+    int64_t evictableSize { 0 };
+    uint64_t maximumBufferSize { 0 };
+    size_t numMediaSamples { 0 };
```

Comment:
Strange that all size are not size_t, probably does not matter.

Reply:
I kept the original size. I am never 100% sure with 32 bits platform support, and size_t being 32 bits on some platform you'll need to cater for overflow.